packages/inspection-capture-web/src/PhotoCapture/PhotoCaptureHUD/PhotoCaptureHUDElementsSight/SightGuideline/SightGuideline.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useMemo, useState } from 'react';
+import { Button } from '@monkvision/common-ui-web';
+import { SightGuideline, MonkLanguage, monkLanguages } from '@monkvision/types';
+import { useTranslation } from 'react-i18next';
+import { usePhotoCaptureHUDButtonBackground } from '../../hooks';
+import { styles } from './SightGuideline.styles';
+import { getVisilityStyle } from '../utils';
+
+/**
+ * Props of the SightGuidelineButton component.
+ */
+export interface SightGuidelineButtonProps {
+  /**
+   * The id of the sight.
+   */
+  sightId: string;
+  /**
+   * A collection of sight guidelines in different language with a list of sightIds associate to it.
+   */
+  sightGuidelines?: SightGuideline[];
+  /**
+   * Boolean indicating whether the sight guideline feature is enabled. If disabled, the guideline text will be hidden.
+   *
+   * @default true
+   */
+  enableSightGuideline?: boolean;
+  /**
+   * Boolean indicating whether the Add Damage feature is enabled. If disabled, the `Add Damage` button will be hidden.
+   *
+   * @default true
+   */
+  enableAddDamage?: boolean;
+}
+
+function getLanguage(language: string): MonkLanguage {
+  const languagePrefix = language.slice(0, 2) as MonkLanguage;
+  return monkLanguages.includes(languagePrefix) ? languagePrefix : 'en';
```

Comment:
export this in a utils function in common package, and also replace it in useObjectTranslation